### PR TITLE
Tag head commit when receiving dispatch event

### PR DESCRIPTION
**Pull request by nmellado**
see https://github.com/STORM-IRIT/Radium-Releases/pull/10

### CODE REVIEW

**Comment by dlyr** [comment]
**CHECK LINEAR HISTORY FAIL**

Please rebase on current up to date master,
without merge commits (e.g. you can squash them).

File: .github/workflows/createTagFromDispatch.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Create tag (notified from dispatch event)
```
Comment (dlyr) [inline .github/workflows/createTagFromDispatch.yml:1]: should the filename be 
create-tag-from-dispatch.yml
to be consistent with the other workflow file naming convention here ?
I see the convention is different on Radium-Release but the same on Radium-Apps ... it's not mandatory, but cleaner to have consistency.

Comment (dlyr) [inline .github/workflows/createTagFromDispatch.yml:1]:
```suggestion
name: Create tag (notified from Radium-Release dispatch event)
```

**Review by dlyr — CHANGES_REQUESTED** [review]

File: .github/workflows/createTagFromDispatch.yml (hunk 1 above)
Comment (nmellado) [reply .github/workflows/createTagFromDispatch.yml:1]: I didn't mention Radium-Release here, as we might have other trigger in the future (does not have to be from Radium-Release)

Comment (nmellado) [reply .github/workflows/createTagFromDispatch.yml:1]: Hum right, I've followed the naming convention of Radium-Releases, but true, it's better to stick to Radium-Engine convention.

Comment (nmellado) [reply .github/workflows/createTagFromDispatch.yml]: Done.

**Review by dlyr — DISMISSED** [review]

**Comment by nmellado** [comment]
Qt is down. Same on Radium-Release.

**Review by dlyr — APPROVED** [review]

**Pull request merged**